api/v1alpha1/openapi.yaml

```diff
@@ -245,77 +227,75 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+        "403":
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
         "404":
           description: NotFound
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
         "500":
-          description: Internal Server Error
+          description: NotFound
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-  /api/v1/agents:
-    get:
+  /api/v1/image:
```

Comment:
Let's have this as `source` subresource?

Reply:
ok